### PR TITLE
Add more methods to the new API

### PR DESCRIPTION
**Pull request by arscan**
Add vread, history, and read_feed methods to new API, and one bugfix on create.

### CODE REVIEW

File: lib/ext/model.rb [hunk 1]
```diff
@@ -29,6 +29,26 @@ def self.read_with_summary(id, summary, client = self.client)
       handle_response client.read(self, id, client.default_format, summary)
     end
 
+    def self.vread(id, version_id, client = self.client)
+      handle_response client.vread(self, id, version_id)
+    end
```
Comment (asross) [inline lib/ext/model.rb]: You could define both instance and class methods
```ruby
def self.vread(id, version_id, client = self.client)
  handle_response client.vread(self, id, version_id)
end

def vread(version_id)
  self.class.vread(id, version_id, client)
end
```

**Comment by asross** [comment]
👍

**Comment by andrequina** [comment]
@arscan take a look at the comments from @asross and send me a know when I should take another look

**Comment by arscan** [comment]
Added a vread instance method per @asross suggestion.  Updated the resource test in plan_executor to use this as well.

**Comment by asross** [comment]
LGTM!

**Pull request merged**